conf/task_defaults/byol.yaml

```diff
@@ -0,0 +1,27 @@
+program:  # These are experiment level arguments
+  overwrite: True
+  data_dir: /datadrive/cvpr_chesapeake_landcover
```

Comment:
Personal directories need to be removed from the default configs, you can always override this in your own config or from the command line.